project/threads/api.py

```diff
@@ -64,11 +65,11 @@ def get_thread(request, thread_id):
     try:
         t = Thread.objects.get(id=thread_id)
         civis = Civi.objects.filter(thread_id=thread_id)
-        req_a = Profile.objects.get(user=request.user)
+        requested_user = request.user
 
         # TODO: move order by to frontend or accept optional arg
         c = civis.order_by("-created")
-        c_scores = [ci.score(req_a.id) for ci in c]
+        c_scores = [ci.score(requested_user.id) for ci in c]

```

Comment:
Can we just use `request.user.id` here directly?
```suggestion
        c_scores = [ci.score(request.user.id) for ci in c]

```

Reply:
@brylie, I'll do. I was editing my PR message to write changes. Could you check it?